q2cli/tools.py

```diff
@@ -330,26 +330,29 @@ def view(visualization_path, index_extension):
                     "and actual data. Use 'qiime tools export' to export only "
                     "the data stored in an Artifact or Visualization, with "
                     "the choice of exporting to different formats.")
-@click.argument('path', type=click.Path(exists=True, file_okay=True,
-                                        dir_okay=False, readable=True))
-@q2cli.option('--output-dir', required=False,
-              type=click.Path(exists=True, file_okay=False, dir_okay=True,
+@q2cli.option('--input-path', required=True,
+              type=click.Path(exists=True, file_okay=True, dir_okay=False,
+                              readable=True),
+              help='Directory where archive should be extracted from')
```

Comment:
How about we match the `input-path` text in `import`?

`Path to file that should be extracted`

Reply:
Yeah, I think that sounds better!